content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst

```diff
@@ -0,0 +1,289 @@
+=========================
+Engineering change orders
+=========================
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+.. |BOMs| replace:: :abbr:`BoMs (Bills of Materials)`
+.. |ECO| replace:: :abbr:`ECO (Engineering Change Order)`
+.. |ECOs| replace:: :abbr:`ECOs (Engineering Change Orders)`
+
+.. _plm/eco:
+
+Utilize *Engineering Change Orders* (*ECOs*) to track and implement changes to products, and
+:ref:`bills of materials <manufacturing/management/bill-configuration>`.
+
+View and create :abbr:`ECOs (engineering change orders)` within each *ECO type*, a project pipeline
+in Gantt view with stages, ensuring progress tracking for collaborators and stakeholders.
```

Comment:
Lines 15-16 have mixed subjects and don't offer a clear definition or an instruction.